api/src/utils/ModelsUtils.ts

```diff
@@ -0,0 +1,12 @@
+import {Fiscalizar} from "../types/models";
+
+export const validateFiscalizar: (obj: any) => obj is Fiscalizar = (obj: any): obj is Fiscalizar => {
```

Comment:
Podemos definir un tipo aca en vez de dejar `any`?

Reply:
En este caso no se puede poner otra cosa que any justamente porque lo que esta haciendo el metodo es indicarle al compilador de ts que cualquier objeto que tenga las caracteristicas validadas es un objeto Fiscalizar (ahora Audit). Por eso el return type es "obj is Fiscalizar".